packages/docs/components/CodePreview/CodePreview.tsx

```diff
@@ -89,7 +89,3 @@ export const CodePreview: React.FC<CodePreviewProps> = (props) => {
     </BigDesign.Box>
   );
 };
-
-CodePreview.defaultProps = {
-  scope: defaultScope,
-};
```

Comment:
Noticed that `defaultScope` was directly used in the following code (shown below), so I think `lines 92 to 95` might not be necessary? Unless I'm missing something 😅 :

```
const [scope, setScope] = useState({ ...defaultScope, ...props.scope });

useEffect(() => {
    setScope({ ...defaultScope, ...props.scope });
  }, [props.scope, setScope]);
```